templates/Program/programComments.html.twig

```diff
@@ -71,9 +83,18 @@
               </a>
             {% endif %}
           </div>
-          <div class="comment-text">
+          <div id="comment-text-wrapper-{{ comment.id }}" class="comment-text">
             <p id="comment-text-{{ comment.id }}">{{ comment.text }}</p>
           </div>
+          <div id="comment-translation-wrapper-{{ comment.id }}" class="comment-translation-wrapper" style="display: none">
+            <div id="comment-translation-credit-wrapper-{{ comment.id }}">
+              <span id="comment-translation-provider-{{ comment.id }}" class="comment-translation-credit">{{ 'programs.provider'|trans({}, 'catroweb') }}</span>
+              <span id="comment-translation-source-language-{{ comment.id }}" class="comment-translation-credit">source</span>
+              <span id="comment-translation-credit-transition-{{ comment.id }}" class="comment-translation-credit">{{ 'programs.creditTransition'|trans({}, 'catroweb') }}</span>
+              <span id="comment-translation-target-language-{{ comment.id }}" class="comment-translation-credit">target</span>
+            </div>
+            <p id="comment-text-translation-{{ comment.id }}" class="comment-text-translation" lang="">Error, comment not translated</p>
```

Comment:
When is the error shown? also don't use non standard abbreviations like `lang`

Reply:
Removed the error statement